src/site/utilities/featureFlags.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs');
+const path = require('path');
+
+// Edit this to add new flags
+const featureFlags = {
+  FEATURE1: 'feature1',
+};
+
+// Edit this to turn flags on or off
+const flagsByBuildtype = {
+  // localhost: [],
+  localhost: [featureFlags.FEATURE1],
+  // vagovdev: [flags.FEATURE1],
+  vagovdev: [],
+  // vagovstaging: [flags.FEATURE1],
+  vagovstaging: [],
+  // vagovprod: [flags.FEATURE1],
+  vagovprod: [],
+};
+
+// Exported feature flag state, which can be used in code as needed
+const enabledFeatureFlags = Object.values(featureFlags).reduce((acc, next) => {
+  acc[next] = flagsByBuildtype[global.buildtype].includes(next);
+  return acc;
+}, {});
+
+// For requiring queries
+const requireWithFeatureFlag = (dirname, filePath) => {
```

Comment:
Basically, we're looking through the enabled flags and seeing if there's a file that's suffixed with the flag name. If so, we'll use that file.